client/src/components/Overview.jsx

```diff
@@ -28,7 +28,8 @@ class Overview extends React.Component {
 
   fetchRestaurantInfo() {
     // const id = window.location.href.split('/')[4];
-    let id = 18611;
+    let id = Math.floor(Math.random() * Math.floor(1000000));
```

Comment:
It seems that you are randomizing the id after a req comes from the client to serve these static assets.  If they are being directed to your service because they clicked on a particular id from another service, this will prevent the incoming id from being read and served to your axios.get req, and instead will be overridden.  Perhaps including this logic elsewhere, to randomize a default id if none is provided, will do what you're looking to accomplish.  In addition, it can be refactored when we serve services through one proxy, to match any agreed-upon default behaviour at a root req.